products/rhel8/profiles/ospp.profile

```diff
@@ -210,6 +210,8 @@ selections:
     - package_abrt-addon-ccpp_removed
     - package_abrt-plugin-sosreport_removed
     - package_abrt-cli_removed
+    - package_libreport-plugin-rhtsupport_removed
+    - package_libreport-plugin-logger_removed
```

Comment:
Does it makes sense to add these two rules when their old equivalents are not selected?
- package_abrt-plugin-logger_removed
- package_abrt-plugin-rhtsupport_removed

Same for the STIG profiles.

Reply:
I believe so, the package is basically the same with a different name if I'm not mistaken. The only caveat for STIG is that their recommendation is to remove all `abrt*` package which does not include these new packages.